frontendGraphApi/src/main/java/com/virtuslab/gitmachete/frontend/graph/api/print/IPrintElementGenerator.java

```diff
@@ -0,0 +1,11 @@
+package com.virtuslab.gitmachete.frontend.graph.api.print;
+
+import java.util.Collection;
+
+import org.checkerframework.checker.index.qual.NonNegative;
+
+import com.virtuslab.gitmachete.frontend.graph.api.print.elements.IPrintElement;
+
+public interface IPrintElementGenerator {
```

Comment:
Two remarks wrt. this package:
1) I think extracting `.elements` subpackage isn't really necessary (doesn't increase clarity)
2) Now that we're pretty much independent (?) from the original IntelliJ APIs here, maybe let's rename this `PrintElement` to sth more adequate (and ideally distinct from `Elements` in `graph.api.elements` package)

Reply:
And his name is...

`RenderPart`

Reply:
And his name is...

`RenderPart`